optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -1052,6 +1067,11 @@ def forward(
         self._compile()
         model_inputs = {"input_ids": input_ids}
 
+        if "attention_mask" in self.input_names:
+            model_inputs["attention_mask"] = (
+                attention_mask if attention_mask is not None else torch.ones(input_ids.shape, dtype=torch.long)
```

Comment:
are you sure we wanna create `torch.ones` here ? it doesn't work for batched prompts (batch of samples with different lengths)

Reply:
yes, I'm sure, this common practice to create tensor of ones in case if attention mask required but not provided from user. Otherwise we need to have access to tokenizer and used pad token, that is not always possible, but in case where input with padding provided - tokenizer returns both input_ids and attention_mask, so it should not be problem to use tokenizer prepared attention_mask

Reply:
removed